src/wallet/wallet2.cpp

```diff
@@ -6220,22 +6257,35 @@ void wallet2::store()
     store_to("", epee::wipeable_string());
 }
 //----------------------------------------------------------------------------------------------------
-void wallet2::store_to(const std::string &path, const epee::wipeable_string &password)
+void wallet2::store_to(const std::string &path, const epee::wipeable_string &password, bool force_rewrite_keys)
 {
   trim_hashchain();
 
+  const bool had_old_wallet_files = !m_wallet_file.empty();
+  THROW_WALLET_EXCEPTION_IF(path.size() >= 4 && path.substr(path.size() - 4) == ".new",
+    error::wallet_internal_error, "New wallet save path cannot end in '.new', that extension is reserved");
+  THROW_WALLET_EXCEPTION_IF(!had_old_wallet_files && path.empty(), error::wallet_internal_error,
+    "Cannot resave wallet to current file since wallet was not loaded from file to begin with");
```

Comment:
This error is being triggered when changing the password of an in-memory wallet, since `change_password` calls this function.

Reply:
How about the latest commit?

Reply:
It's fixed with the last commit.